### PR TITLE
BLorient539

### PR DESCRIPTION
**Pull request by thea-m**
https://github.com/BetaMasaheft/Documentation/issues/1447

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient539.xml [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">227</measure>
+                                    <dimensions type="outer" unit="in">
+                                        <height>10.25</height>
+                                        <width>9</width>
+                                    </dimensions>
+                                </extent>
+                                <condition key="deficient">Mutilated. Several leaves have been torn out between
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient539.xml:155]: What is mutilated?

File: LondonBritishLibrary/orient/BLorient539.xml [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+                                
+                                <msItem xml:id="ms_i1.1.3">
+                                    <locus from="175v"/>
+                                    <title type="complete" ref="LIT3985Mazmura#SongOfSongs"/>
+                                </msItem>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.2">
+                                <locus from="184v"/>
+                                <title type="complete">Prayers to Mary and to Christ</title>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient539.xml:71]: No ID here?

File: LondonBritishLibrary/orient/BLorient539.xml [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+                                    <q xml:lang="gez"><seg part="I">እምሥርወ፡ ዕሴይ፡ እስመ፡ ሠረፅከ፡ ወእማኅፀና፡ ለሩት። ናስተበፅዓከ፡ ዘልፈ፡ በንባበ፡ ዝንቱ፡ ስብሐት። 
+                                        ኦብፁዕ፡ ንጉሠ፡ እስራኤል፡ ዳዊት። ሶበ፡ ተከሥተ፡ ወተረከበ፡ ባሕርየ፡ ውሣጣይ፡ ሕይወት። እመዝገበ፡ ከርሣ፡ ለማርያም፡ ዘለከ፡ ሕይወት። ሰላም፡ ለዝክረ፡ 
+                                        ስምከ፡ ውስተ፡ ረቀ፡ መዝሙር፡ ዘተዌደደ።
+                                   </seg></q>
+                                 </item>
+                                
+                                <item xml:id="a2">
+                                    <locus target="#226v"/>
+                                    <desc type="OwnershipNote">A note gave the name of a later owner, for which that of 
+                                        <persName ref="PRS13080Isayeyyas"/> has been substituted.</desc>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient539.xml:196]: The comment on the owner is difficult to understand.

File: LondonBritishLibrary/orient/BLorient539.xml [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+                                    <foreign xml:lang="gez">ክርስቶስ></foreign>, which has been changed into 
+                                        <persName ref="PRS13079HellaweK"/>.</desc>
+                                </item>
+                                
+                                <item xml:id="e3">
+                                    <locus target="#2v"/>
+                                    <desc>The names of <persName ref="PRS13082KidanaM"/> and <persName ref="PRS13080Isayeyyas"/>
+                                        have been substituted for the 
+                                        erased names originally mentioned in the manuscript.</desc>
+                                </item>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient539.xml:224]: sorry, th description is not very clear to me: those names are substituitions for the original erased names?

File: LondonBritishLibrary/orient/BLorient539.xml (hunk 1 above)
Comment (thea-m) [reply LondonBritishLibrary/orient/BLorient539.xml:155]: I copied the catalogue. I think it means material loss.

File: LondonBritishLibrary/orient/BLorient539.xml (hunk 2 above)
Comment (thea-m) [reply LondonBritishLibrary/orient/BLorient539.xml:71]: No, I think that there is not enough information here to assign an ID.

File: LondonBritishLibrary/orient/BLorient539.xml (hunk 3 above)
Comment (thea-m) [reply LondonBritishLibrary/orient/BLorient539.xml:196]: I agree. But there is not more information, so I see nothing we can do here without adding our own interpretation.

File: LondonBritishLibrary/orient/BLorient539.xml (hunk 4 above)
Comment (thea-m) [reply LondonBritishLibrary/orient/BLorient539.xml:224]: yes, please feel free to reformulate if you come up with something clearer.
The names originally mentioned in the manuscript have been erased, and substituted by...?

Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient539.xml:224]: Yes, I think that would be clearer, at least for me :)

Comment (thea-m) [reply LondonBritishLibrary/orient/BLorient539.xml:223]: thank you!

**Pull request merged**